PhysicsTools/NanoAOD/python/jetsAK4_Puppi_cff.py

```diff
@@ -301,29 +289,76 @@ def nanoAOD_refineFastSim_puppiJet(process):
             btagUParTAK4Brefined   = cms.InputTag("puppiJetRefineNN","btagUParTAK4Brefined"),
             btagUParTAK4CvBrefined = cms.InputTag("puppiJetRefineNN","btagUParTAK4CvBrefined"),
             btagUParTAK4CvLrefined = cms.InputTag("puppiJetRefineNN","btagUParTAK4CvLrefined"),
-            btagUParTAK4QvGrefined = cms.InputTag("puppiJetRefineNN","btagUParTAK4QvGrefined"),
-        )
+            btagUParTAK4QvGrefined = cms.InputTag("puppiJetRefineNN","btagUParTAK4QvGrefined"),)
```

Comment:
previous version of these lines is preferable (future edits become easier, otherwise need to move closing parenthesis every time another line is added)